CorsixTH/Src/random.c

```diff
@@ -180,28 +173,21 @@ static int l_randomdump(lua_State *L)
   the random number generator state from a string previously generated from
   math.randomdump().
 */
-static int l_randomseed(lua_State *L)
-{
-    if(lua_type(L, 1) == LUA_TSTRING)
-    {
+static int l_randomseed(lua_State* L) {
+    if (lua_type(L, 1) == LUA_TSTRING) {
         int i;
         size_t len;
-        const char *data = lua_tolstring(L, 1, &len);
-        if(len != N * 4 + 2)
-            luaL_argerror(L, 1, "Seed string wrong length");
-        for(i = 0; i < N; ++i)
-            mt[i] = ((uint32_t*)data)[i];
+        const char* data = lua_tolstring(L, 1, &len);
+        if (len != N * 4 + 2) luaL_argerror(L, 1, "Seed string wrong length");
+        for (i = 0; i < N; ++i) mt[i] = ((uint32_t*)data)[i];
```

Comment:
Many function calls are split in a lot of lines, yet this is allowed. While I like this compact style, not sure if it fits in the general idea of the used style.